examples/test-app/utils/validators.js

```diff
@@ -1,24 +1,43 @@
 const Joi = require('joi');
+const { first } = require('lodash');
 
 const defaultMin = 0;
 const defaultMax = 30;
-const isString = (min = defaultMin, max = defaultMax, required = true) => {
+const isString = ({
+  language = {},
+  min = defaultMin,
+  max = defaultMax,
+  required = true
+}) => {
   const validator = Joi.string().alphanum()
     .min(min)
     .max(max);
   if (required) {
     validator.required();
```

Comment:
For this to work as you expect you will have to set the validator on itself. 
For example: `validator = validator.required();`

Or as the docs say: 

"Note that joi schema objects are immutable which means every additional rule added (e.g. .min(5)) will return a new schema object."